url.bs

```diff
@@ -864,9 +864,9 @@ It is initially the empty string.
 <p>A  <a for=/>URL</a>'s <dfn export for=url id=concept-url-port>port</dfn> is either
 null or a 16-bit unsigned integer that identifies a networking port. It is initially null.
 
-<p>A <a for=/>URL</a>'s <dfn export for=url id=concept-url-path>path</dfn> is a list of zero or more
-<a>ASCII strings</a> holding data, usually identifying a location in hierarchical form. It is
-initially the empty list.
+<p>A <a for=/>URL</a>'s <dfn export for=url id=concept-url-path>path</dfn> is a <a>list</a> of zero
```

Comment:
This isn't linking for some reason: https://url.spec.whatwg.org/branch-snapshots/annevk/infra-wip/#concept-url-path